commands/devboard.rb

```
@@ -0,0 +1,40 @@
+require 'json'
+require 'net/http'
+require 'net/https'
+
+# Get data from Devboard
+def devboard_command
+  uri =  URI.parse($SETTINGS["DEVBOARD_JSON"])
+
+  http = Net::HTTP.new(uri.host, uri.port)
+  http.use_ssl = true if uri.scheme == "https" 
+  request = Net::HTTP::Get.new(uri.request_uri)
+  request.basic_auth $SETTINGS["DEVBOARD_API_USER"], $SETTINGS["DEVBOARD_API_SECRET"]
+
+  response = http.request(request)
+
+  # Proper Credentials met
+  message = ""
+  if response.code == "200"
```

What is the code isn't a 200? Chatbot will fail silently? How about an error message in the channel instead like "Unable to connect to Devboard."